src/Sulu/Component/Content/Document/Subscriber/ResourceSegmentSubscriber.php

```diff
@@ -207,4 +245,51 @@ private function persistRoute(ResourceSegmentBehavior $document)
     {
         $this->rlpStrategy->save($document, null);
     }
+
+    /**
+     * Recreates the routes for the destination document with the data from the source document.
+     *
+     * Note that both documents can be the same (e.g. happening on a move). If the second parameter is omitted it has
+     * the same value as the first one.
+     *
+     * @param object $sourceDocument
+     * @param object $destinationDocument
+     */
+    private function recreateRoutes($sourceDocument, $destinationDocument = null)
+    {
+        $destinationDocument = $destinationDocument ?: $sourceDocument;
+
+        if (
+            !$sourceDocument instanceof ResourceSegmentBehavior
+            || !$destinationDocument instanceof  ResourceSegmentBehavior
+        ) {
+            return;
+        }
+
+        $locales = $this->documentInspector->getLocales($destinationDocument);
+        $webspaceKey = $this->documentInspector->getWebspace($destinationDocument);
+        $sourceUuid = $this->documentInspector->getUuid($sourceDocument);
+        $destinationUuid = $this->documentInspector->getUuid($destinationDocument);
+        $destinationParentUuid = $this->documentInspector->getUuid(
+            $this->documentInspector->getParent($destinationDocument)
+        );
+
+        foreach ($locales as $locale) {
+            $localizedDocument = $this->documentManager->find($destinationUuid, $locale);
+
+            if ($localizedDocument->getRedirectType() !== RedirectType::NONE) {
```

Comment:
no ignore it also before shadow wouldnt be ignored (: was a little bit confused from deleted and added code!